documentation_builder/plot_helper.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from matplotlib.pyplot import figure, xlim, ylim, gca, arrow, text, scatter
-from mpl_toolkits.axes_grid.axislines import SubplotZero
+from mpl_toolkits.axisartist.axislines import SubplotZero
```

Comment:
Is that module change from a particular version? Then we should add a minimum version to the requirements.

Reply:
Done.